helperFunctions.js

```diff
@@ -2,20 +2,20 @@ const nodemailer = require('nodemailer'); //Sending Email Conformations and car
 
 async function sendConfirmation(userEmail, objectID) {
   let transporter = nodemailer.createTransport({
-    service: 'gmail',
+    host: 'smtp.office365.com',
+    port: 587,
+    secure: false, // Set to true if you want to use TLS (not recommended for port 587)
     auth: {
-      user: 'ronshaked07@gmail.com',
-      pass: 'sknf irxh yhaa uhjq'
-    }
+      user: 'donotreply@automatch.dev', // Your custom email address
+      pass: 'e8$6rRgQVd#4tF2', // Your Microsoft 365 email password
+    },
   });
-
-  const confirmationLink = `http://localhost:8080/confirm/user?objectid=${objectID}`;
+  
   const details = {
-    from: 'ronshaked07@gmail.com',
+    from: 'donotreply@automatch.dev', // Your custom email address
     to: userEmail,
-    subject: 'Confirmation Email',
-    text: `Click the link below to confirm your account:\n${confirmationLink}`,
-    html: `Click the link below to confirm your account:<br><a href="${confirmationLink}">Confirm your account</a>`
+    subject: 'AutoMatch - Verify your Account',
+    text: `Click here to verify your account: http://localhost:8080/confirm/user?objectid=${objectID}`
```

Comment:
Make it so it says "Confirm Account" instead of a sketchy url, (Feedback I got from CS Forum people)

Reply:
Done